tests/test_fassert_randomize.py

```diff
@@ -0,0 +1,120 @@
+import unittest
```

Comment:
Can you merge these functions into `test_assert_randomize.py` instead of making a new` test_fassert_randomize.py`? I think it makes maintenance easier when we have a 1-to-1 mapping between file names in the `duvm/` folder and tests in the `test/` folder. E.g., everything in `assert_randomize.py` is tested by `test_assert_randomize.py`

Reply:
pushed up